### ISSUE TITLE
[Android] App crashes if not using Ktor platform client 

### ISSUE THREAD

**Issue by Monabr** [Foso/Ktorfit#720]
### Ktorfit version

2.1.0

### What happened and how can we reproduce this issue?

So the docs states there is no need in additional configuration.

If I just use

```
            implementation(libs.ktorfit.lib)
            implementation(libs.ktor.client.serialization)
            implementation(libs.ktor.client.content.negotiation)
            implementation(libs.ktor.serialization.kotlinx.json)
```

and start the app - I get the error:

```
java.lang.NoClassDefFoundError: Failed resolution of: Lio/ktor/client/plugins/HttpTimeout;
```

If I add platform clients (okhttp and darwin)  and use them - the app works fine

### What did you expect to happen?

Docs be accurate or there is was not bug

**Comment by geworfener** [comment]
Having the same problem, on iOS only though:

`Caused by: kotlin.IllegalStateException: Failed to find HttpClientEngineContainer. Consider adding [HttpClientEngine] implementation in dependencies.`

**Comment by Foso** [comment]
It's fixed with Ktorfit 2.2.0
